pages/app-deploy/laravel/tips.js

```diff
@@ -604,36 +602,22 @@ $ffmpeg = FFMpeg::create([
       </a>{" "}
       را مطالعه کنید.
     </p>
-    <h3 id="using-ignition-and-faker-packages">
-      استفاده از پکیج‌های Ignition و Faker
-    </h3>
+    <h3 id="using-ignition-and-faker-packages">نصب پکیج‌های Dev</h3>
     <p>
-      شما برای استفاده از پکیج‌های{" "}
-      <a href="https://github.com/facade/ignition" target="_blank">
-        Ignition
-      </a>{" "}
-      و{" "}
-      <a href="https://github.com/FakerPHP/Faker" target="_blank">
-        Faker
-      </a>{" "}
-      در لیارا باید ابتدا این پکیج‌ها را از بخش{" "}
+      شما برای نصب پکیج‌هایی که در بخش
       <span className="code">require-dev</span> فایل{" "}
-      <span className="code">composer.json</span> به بخش{" "}
-      <span className="code">require</span> انتقال دهید:
+      <span className="code">composer.json</span> قرار دارد، باید تنظیمات زیر را
```

Comment:
Including the package names will help users to find this section by searching the keywords:

```suggestion
      <span className="code">composer.json</span> قرار دارد (مانند Faker و Ignition)، باید تنظیمات زیر را
```